recipe/meta.yaml

```
@@ -87,38 +82,38 @@ requirements:
     - zlib
     # requirements specified by the package itself, see
     # github.com/tensorflow/tensorflow/blob/v{{ version }}/tensorflow/tools/pip_package/setup.py
-    - absl-py >=0.10,<1
-    - astunparse >=1.6.3,<1.7
-    - python-flatbuffers >=1.12,<2
-    - google-pasta >=0.2,<1
-    - h5py >=3.1.0,<3.2
-    - keras-preprocessing >=1.1.2,<1.2
-    - numpy 1.19.*
-    - opt_einsum >=3.3,<3.4
+    - absl-py >=0.4.0
+    - astunparse >=1.6.0
+    - gast >=0.2.1,<0.5.0
+    - google-pasta >=0.1.1
+    - h5py >=2.9.0
+    - keras-preprocessing >=1.1.1
+    - numpy
+    - opt_einsum >=2.3.2
     - protobuf >=3.9.2
-    # 2021/12/05: hmaarrfk
-    # While tensorflow has an upper bound on six
-    # six has been highly backward compatible
-    # We remove the upper bound so that we can be more compatible with
-    # The rest of the conda-forge ecosystem
-    - six >=1.15
-    - termcolor >=1.1,<1.2
-    - typing_extensions >=3.7.4,<3.8
-    - wheel >=0.35,<1
-    - wrapt >=1.12.1,<1.13
-    # upstream wants exact pin for gast
-    - gast ==0.4.0
+    - python-flatbuffers >=1.12,<3
+    - six >=1.12
+    - termcolor >=1.1.0
+    - typing_extensions >=3.6.6
+    - wrapt >=1.11.0
     # TF-API needs to move in sync
+    # XXX: actually, TF 2.7.0 still demands tb 2.6
     - tensorboard >=2.6,<2.7
-    - keras >=2.6,<2.7
+    - keras >=2.7,<2.8
 
 outputs:
+  # as of tensorflow 2.7.0 we need the tensorlfow base package
+  # to break circular dependency when buliding tensorflow extra packages
+  # This annoying is broken upstream by the fact that they have to bootstrap their builds
```

```suggestion
  # as of tensorflow 2.7.0 we need the tensorflow base package
  # to break circular dependency when buliding tensorflow extra packages
  # This annoying circularity is broken upstream by the fact that they have to bootstrap their builds
```

seems there's a word missing.